tools/quake3/common/polylib.c

```diff
@@ -96,7 +96,7 @@ winding_accu_t *AllocWindingAccu( int points ){
 			c_peak_windings = c_active_windings;
 		}
 	}
-	s = sizeof(*w) + (points > 4 ? sizeof(vec3_accu_t) * (points - 4) : 0);
+	s = sizeof( *w ) + points * sizeof( *w->p );
```

Comment:
same here, someone had the idea to rewrite it this way instead:

```c
s = sizeof( *w ) + ( points ? sizeof( w->p[0] ) * ( points - 1 ) : 0 );
```

what can be the meaning of that trick?

Reply:
Yes, again check out how the type is defined different: In q3map2, we now have:

    typedef struct
    {
        int numpoints;
        vec3_accu_t p[];
    } winding_accu_t;

Presumably in their code the last member is `p[1]`, not `p[]`?

Reply:
Are flexible array members supported in MSVC?

Reply:
@ensiform: [looks like it](https://godbolt.org/g/TRNv34).